### PR TITLE
feat(oui-tabs): add tabs component

### PR DESCRIPTION
**Pull request by AxelPeter**
UK-62

Related to
* https://github.com/ovh-ux/ovh-ui-kit/pull/311
* https://github.com/ovh-ux/ovh-ui-kit-documentation/pull/173

### CODE REVIEW

File: packages/oui-tabs/src/item/tabs-item.controller.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    setActive () {
+        this.tabsCtrl.activeId = this.id;
+    }
+
+    getNodeIndex () {
+        let i = 0;
+        let item = this.$element[0];
+
+        // Return DOM node index
+        while (item.previousSibling) {
```
Comment (Jisay) [inline packages/oui-tabs/src/item/tabs-item.controller.js]: I find strange to see DOM manipulations and DOM acces in controller. This should be done in link function of a directive as described in angularJS best practice, no?

Comment (AxelPeter) [reply packages/oui-tabs/src/item/tabs-item.controller.js]: It's more DOM navigation than DOM manipulation, but we could put that part in $postLink :)

File: packages/oui-tabs/src/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
+describe("ouiSwitch", () => {
```
Comment (marie-j) [inline packages/oui-tabs/src/index.spec.js]:
```suggestion
describe("ouiTabs", () => {
```
? 😄

Comment (AxelPeter) [reply packages/oui-tabs/src/index.spec.js]: Bad copy/paste :)

File: packages/oui-tabs/README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    </oui-tabs-item>
+    <oui-tabs-item heading="Dolor sit amet" checked="$ctrl.checked3">
+        <oui-button on-click="$ctrl.checked3 = !$ctrl.checked3">Toggle check mark of tab 3</oui-button>
+    </oui-tabs-item>
+</oui-tabs>
+```
+
+## API
+
+### oui-tabs-item
```
Comment (antleblanc) [inline packages/oui-tabs/README.md]:
```suggestion
### oui-tabs
```

Comment (AxelPeter) [reply packages/oui-tabs/README.md]: Good catch :)

**Pull request merged**